doc/source/serve/production-guide/index.md

```diff
@@ -0,0 +1,52 @@
+(serve-in-production)=
+
+# Production Guide
+
+The recommended way to run Ray Serve in production is on Kubernetes using the [KubeRay](kuberay-quickstart) [RayService] custom resource.
+The RayService custom resource automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
+If you're not running on Kubernetes, you can also run Ray Serve on a Ray cluster directly using the Serve CLI.
+
+This section should help you:
+
+- understand the [Serve config file format](serve-in-production-config-file).
+- understand how to deploy on VMs using the [Serve CLI](serve-in-production-deploying).
+- understand how to deploy on Kubernetes using a [`KubeRay` `RayService`](serve-in-production-kubernetes).
+- understand how to [monitor running Serve applications](serve-monitoring).
+
+(serve-in-production-example)=
+
+## Working example: FruitStand application
+
+Throughout the production guide, we will use the following Serve application as a working example.
+The application takes in requests containing a list of two values, a fruit name and an amount, and returns the total price for the batch of fruits.
+
+```{literalinclude} ../serve/doc_code/production_fruit_example.py
+:language: python
+:start-after: __fruit_example_begin__
+:end-before: __fruit_example_end__
+```
+
+This code is saved in the `fruit.py` file and the final deployment graph that we want to deploy is assigned to the `deployment_graph` variable.
```

Comment:
```suggestion
To follow along, save the code in a file named `fruit.py`. We will deploy the `deployment_graph` defined by the script.
```

It would be nice to also include a wget or git clone command here that they can use.